cmd/route-emitter/main_test.go

```diff
@@ -135,10 +135,28 @@ var _ = Describe("Route Emitter", func() {
 						PrivateInstanceId: instanceKey.InstanceGuid,
 					})))
 				})
+
+				Context("and the instance has a route with a route service url", func() {
+					BeforeEach(func() {
+						routes := newRoutes(hostnames, containerPort, "https://awesome.com")
+						desiredLRP.Routes = routes
+					})
+
+					It("emits routes immediately including the route service url", func() {
```

Comment:
Adding the route service url to the top level before each, line 62, means we would change all the tests to expect the new field.  The new field is optional and we might end up adding another context to test when the route service url is not present. The reason why we added a context to test for route service url explicitly is because it's an optional field.  Let us know what you think.